package.json

```diff
@@ -1103,6 +1095,20 @@
                     "description": "%mssql.enableRichExperiencesDoNotShowPrompt.description%",
                     "scope": "application"
                 },
+                "mssql.useLegacyConnectionExperience": {
```

Comment:
I had to add these for the playwright tests to work.  Figuring out why was a bit of a journey:

* The playwright tests run VS Code with a temporary user profile.
* When VS Code runs with a different user profile, only settings that are scoped to the application are read/used.  Anything else is ignored.
* Settings not registered in `package.json` are defaulted to the user scope.
* Therefore, "hidden"/unregisted flags (like I was using for the legacy connection UI) aren't able to be used with the playwright setup.

My solution is to add them with the application scope, but mark them as deprecated so that they don't show up in the settings UI or JSON autocomplete.